File: pset6/sentimental-mario-less/mario.py
# Print mario pyramids
from cs50 import get_int

while True:
    # Prompt for height
    height = get_int('Height: ')
    if height <= 0 or height == 9:
        continue
    else:
        # Draw a pyramid using user height
        # For each raw
        for i in range(height):
            # Draw a space to flip the pyramid from left to right
            print("", end=" " * (height - i - 1))
            # For each column
            for j in range(i + 1):
                # Draw a hash blocks with a formila x - 1
                print("#", end="" * (i - 1))
            # New line
            print()
        break

File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        //Prompt user for height
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // For each row
    for (int i = 0; i < n; i++)
    {
        // Add space for it to be on right
        for (int s = 0; s < n - i - 1; s++)
        {
            printf(" ");
        }
        // For each column
        for (int j = 0; j <= i; j++)
        {
            //add brick
            printf("#");
        }
        //move to next line
        printf("\n");
    }
}